### ISSUE TITLE
Unable to stub patch operations with Open API

### ISSUE THREAD

**Issue by tpwst555** [specmatic/specmatic#547]
We are unable to stub patch operations using an OpenAPI spec.

Steps to reproduce:

1) Provide spec to specmatic including a patch operation:
```
paths:
  /myResources/{myResourceId}:
    patch:
      operationId: patchMyResource
      parameters:
      - name: myResourceId
        in: path
        required: true
        schema:
          type: integer
          format: int64
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/MyResource'
        required: true
      responses:
        "200":
          description: OK
          content:
            '*/*':
              schema:
                $ref: '#/components/schemas/MyResourceResp'
```

2) Submit request to stub server for /myResources/1.

Stub server responds with 
```
"No matching REST stub or contract found for method PATCH and path /myResources/1 (assuming you're looking for a REST API since no SOAPAction header was detected)
```

Expected behavior: Server responds with a mock response.

Specmatic v0.55.0.

Notes:
PATCH method is excluded from the openApiOperations map in OpenApiSpecification.kt.

**Comment by harikrishnan83** [comment]
@tpwst555 Thanks for reporting the issue. We will take a look.

**Comment by harikrishnan83** [comment]
@tpwst555 Thanks for taking a peek at the code and pointing out the exact change. We would also be happy to accept any Pull Requests next time if you have any additions to make.

I have added support for http patch method in OpenAPI. This should be part of our next release. Meanwhile if you have any questions about Specmatic please feel free to reach us at [specmatic.in](https://specmatic.in/contact-us/)

**Comment by harikrishnan83** [comment]
@tpwst555 The fix for this is available as part of release [0.56.0](https://github.com/znsio/specmatic/releases/tag/0.56.0). Closing this issue. Please feel free to re-open if you notice any issues.
